### ISSUE TITLE
Validation not working

### ISSUE THREAD

**Issue by chadidi** [jifalops/datetime_picker_formfield#40]
validation fails at this line https://github.com/jifalops/datetime_picker_formfield/blob/1317954c09c6c3a07be315b4348282b4a1e8944b/lib/datetime_picker_formfield.dart#L300

with this error: 
```
I/BufferQueueProducer(28588): [SurfaceTexture-0-28588-0](this:0x77d05f7000,id:0,api:1,p:350,c:28588) queueBuffer: slot 0 is dropped, handle=0x77df643560
I/BufferQueueProducer(28588): [SurfaceTexture-0-28588-0](this:0x77d05f7000,id:0,api:1,p:350,c:28588) queueBuffer: fps=1.03 dur=1940.99 max=1002.44 min=938.55
I/flutter (28588): ══╡ EXCEPTION CAUGHT BY GESTURE ╞═══════════════════════════════════════════════════════════════════
I/flutter (28588): The following NoSuchMethodError was thrown while handling a gesture:
I/flutter (28588): The getter '_value' was called on null.
I/flutter (28588): Receiver: null
I/flutter (28588): Tried calling: _value
I/flutter (28588):
I/flutter (28588): When the exception was thrown, this was the stack:
I/flutter (28588): #0      Object.noSuchMethod  (dart:core-patch/object_patch.dart:50:5)
I/flutter (28588): #1      DateTime.difference  (dart:core-patch/date_patch.dart:201:54)
I/flutter (28588): #2      _NewEventState._buildPage.<anonymous closure>
package:beacon/pages/newEvent.dart:246
I/flutter (28588): #3      _DateTimePickerTextFormFieldState.build.<anonymous closure>
package:datetime_picker_formfield/datetime_picker_formfield.dart:300
I/flutter (28588): #4      FormFieldState._validate (package:flutter/src/widgets/form.dart)
I/flutter (28588): #5      FormFieldState.validate.<anonymous closure> 
package:flutter/…/widgets/form.dart:363
I/flutter (28588): #6      State.setState 
package:flutter/…/widgets/framework.dart:1117
I/flutter (28588): #7      FormFieldState.validate 
package:flutter/…/widgets/form.dart:362
I/flutter (28588): #8      FormState._validate 
package:flutter/…/widgets/form.dart:202
I/flutter (28588): #9      FormState.validate 
package:flutter/…/widgets/form.dart:196
I/flutter (28588): #10     _NewEventState._buildPage.<anonymous closure>
package:beacon/pages/newEvent.dart:354
I/flutter (28588): #11     _InkResponseState._handleTap 
package:flutter/…/material/ink_well.dart:511
I/flutter (28588): #12     _InkResponseState.build.<anonymous closure> 
package:flutter/…/material/ink_well.dart:566
I/flutter (28588): #13     GestureRecognizer.invokeCallback 
package:flutter/…/gestures/recognizer.dart:166
I/flutter (28588): #14     TapGestureRecognizer._checkUp 
package:flutter/…/gestures/tap.dart:240
I/flutter (28588): #15     TapGestureRecognizer.acceptGesture 
package:flutter/…/gestures/tap.dart:211
I/flutter (28588): #16     GestureArenaManager.sweep 
package:flutter/…/gestures/arena.dart:156
I/flutter (28588): #17     _WidgetsFlutterBinding&BindingBase&GestureBinding.handleEvent 
package:flutter/…/gestures/binding.dart:225
I/flutter (28588): #18     _WidgetsFlutterBinding&BindingBase&GestureBinding.dispatchEvent 
package:flutter/…/gestures/binding.dart:199
I/flutter (28588): #19     _WidgetsFlutterBinding&BindingBase&GestureBinding._handlePointerEvent 
package:flutter/…/gestures/binding.dart:156
I/flutter (28588): #20     _WidgetsFlutterBinding&BindingBase&GestureBinding._flushPointerEventQueue 
package:flutter/…/gestures/binding.dart:102
I/flutter (28588): #21     _WidgetsFlutterBinding&BindingBase&GestureBinding._handlePointerDataPacket 
package:flutter/…/gestures/binding.dart:86
I/flutter (28588): #25     _invoke1  (dart:ui/hooks.dart:233:10)
I/flutter (28588): #26     _dispatchPointerDataPacket  (dart:ui/hooks.dart:154:5)
I/flutter (28588): (elided 3 frames from package dart:async)
I/flutter (28588):
I/flutter (28588): Handler: onTap
I/flutter (28588): Recognizer:
I/flutter (28588):   TapGestureRecognizer#5b0e1(debugOwner: GestureDetector, state: ready, won arena, finalPosition:
I/flutter (28588):   Offset(156.9, 628.7), sent tap down)
I/flutter (28588): ════════════════════════════════════════════════════════════════════════════════════════════════════
```

**Comment by jifalops** [comment]
Can you post the code that you used?

**Comment by jifalops** [comment]
I'm getting a similar issue on v0.3.0

**Comment by jifalops** [comment]
This is fixed in v0.3.1.
